documentation/modules/ROOT/pages/operations/embedded.adoc

```diff
@@ -316,6 +316,36 @@ The default is a periodic commity policy based upon time intervals.
 |The Converter class that should be used to serialize and deserialize value data for offsets. The default is JSON converter.
 |=======================
 
+[[database-history-properties]]
+=== Database history properties
+
+Some of the connectors also require an additional set of properties that configure the database schema history:
+
+* MySQL
+* SQL Server
+* Oracle
+* Db2
+
+Without configuring the database history, the connectors will refuse to start.
```

Comment:
Is this true actually, or will it try and use Kafka on localhost by default (and refuse to start if that fails)? If this is actually what happens, I'd be very much in favour of doing the small code change required for exactly doing what you document here, because that default doesn't seem reasonable.

Reply:
@gunnarmorling  Re-worked to reflect this fact.